tests/test_utils.py

```diff
@@ -0,0 +1,95 @@
+from unittest import TestCase
+
+from graph.elements.edge import Edge
+from graph.elements.hyperedge import HyperEdge
+from graph.elements.node import Node
+from graph.hypergraph import HyperGraph
+from utils import check, _find_subgraphs
```

Comment:
In python with underscore, you mark methods private and protected. They shouldn't be imported like that. If you see no other way than using it, then change it name to be styleguide valid (`find_subgraph`)